opentasks-provider/src/androidTest/java/org/dmfs/provider/tasks/TaskProviderTest.java

```diff
@@ -199,7 +232,111 @@ public void testInsertTaskWithStartAndDue()
                 new Assert<>(task, new TimeData(start, due)),
                 new AssertRelated<>(new InstanceTable(mAuthority), Instances.TASK_ID, task, new AllOf(
                         new EqArg(Instances.INSTANCE_START, start.getTimestamp()),
+                        new EqArg(Instances.INSTANCE_ORIGINAL_TIME, start.getTimestamp()),
+                        new EqArg(Instances.INSTANCE_DUE, due.getTimestamp()),
+                        new EqArg(Instances.INSTANCE_START_SORTING, start.getInstance()),
```

Comment:
`getInstance()` returns the instance date and time values (i.e. year, month, day, hour, minutes and seconds) packed into a single long. These vales are represented in the time zone of the event (or local time zone for floating events). In contrast to `getTimestamp` this allows correct sorting if all-day and non-all-day events are mixed. The `*_SORTING` columns are not meant to be evaluated other than using them for sorting. There is no guarantee about the "format" of this long value and it might change with every new version.